test/test_frame_2d.py

```diff
@@ -1,22 +1,44 @@
+import numpy as np
 import pytest
 
 
 @pytest.mark.physical_camera
 def test_image_context_manager(physical_camera_frame_2d):
     import zivid
 
-    with physical_camera_frame_2d.image_rgba() as image:
-        assert image is not None
-        assert isinstance(image, zivid.Image)
+    with physical_camera_frame_2d.image_rgba() as image_rgba:
+        assert image_rgba is not None
+        assert isinstance(image_rgba, zivid.Image)
+
+    with physical_camera_frame_2d.image_bgra() as image_bgra:
+        assert image_bgra is not None
+        assert isinstance(image_bgra, zivid.Image)
 
 
 @pytest.mark.physical_camera
 def test_image(physical_camera_frame_2d):
```

Comment:
You ran through all these tests with a physical camera right? 
(important since they are not run in the CI)

Reply:
I did not run all again because I didn't have access to physical camera. I'll ask someone for assistance.